compression/zinfo.c

```diff
@@ -495,19 +494,19 @@ unsigned get_blob_size(struct gzip_zinfo* index)
         return 0;
     }
 
-    unsigned size = decode_int32(index->size); 
+    unsigned size = decode_int32(index->size);
+    if (decode_int32(index->version) == 1) {
+        size--;
+    }
```

Comment:
Perhaps a short comment here, to set a precedent that we explain the version differences where they exist in code?